# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/xfce4-mixer/devfs.patch
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./panel-plugin/vc_oss.c.orig	2003-07-18 16:08:27.000000000 +0300
+++ ./panel-plugin/vc_oss.c	2003-08-14 13:55:03.000000000 +0300
@@ -74,7 +74,7 @@
 
 #define MAX_AMP 100
 
-static char dev_name[PATH_MAX] = { "/dev/mixer" };
+static char dev_name[PATH_MAX] = { "/dev/sound/mixer" };
 
 static int mixer_handle = -1;
 static int devmask = 0;                /* Bitmask for supported mixer devices */
